src/algorithms/standard/idct.cpp

```diff
@@ -29,9 +29,10 @@ const char* IDCT::category = "Standard";
 const char* IDCT::description = DOC("This algorithm computes the Inverse Discrete Cosine Transform of an array.\n"
 "It can be configured to perform the inverse DCT-II form, with the 1/sqrt(2) scaling factor for the first coefficient or the inverse DCT-III form based on the HTK implementation.\n"
 "\n"
-"IDCT can be use to compute smoothed Mel Bands. In order to do this: \n "
+"IDCT can be used to compute smoothed Mel Bands. In order to do this: \n "
 "- compute MFCC\n"
 "- smoothedMelBands = 10^(IDCT(MFCC)/20) \n"
+"Note: The second step assumes that the default 'logType' were used to compute MFCCs, otherwise that formula should be changed in order to be consistent.\n"
```

Comment:
explicitly write the 'logType' value  just in case we will change default value in future